lib/phoenix/live_dashboard/live/processes_live.ex

```diff
@@ -32,71 +32,66 @@ defmodule Phoenix.LiveDashboard.ProcessesLive do
-                  )
-                </th>
-                <th class="border-top-0">
-                  MsgQ
-
-                  (
-                    <%= sort_link(@socket, @params, :message_queue_len, :asc) %> |
-                    <%= sort_link(@socket, @params, :message_queue_len, :desc) %>
-                  )
-                </th>
-                <th class="border-top-0">Current function</td>
-              </tr>
-            </thead>
-            <tbody>
-              <%= for process <- @processes, list_pid = encode_pid(process[:pid]) do %>
-                <tr phx-click="show_info" phx-value-pid="<%= list_pid %>" phx-page-loading class="<%= row_class(process, @pid) %>">
-                  <td><%= list_pid %></td>
-                  <td><%= format_name_or_initial_call(process[:name_or_initial_call]) %></td>
-                  <td><%= process[:memory] %></td>
-                  <td><%= process[:reductions] %></td>
-                  <td><%= process[:message_queue_len] %></td>
-                  <td><%= SystemInfo.format_call(process[:current_function]) %></td>
+          </div>
+          <div class="col-auto">
+            processes out of <%= @total %>
+          </div>
+        </div>
+      </form>
+
+      <%= if @pid do %>
+        <%= live_modal @socket, ProcessInfoComponent,
+          id: @pid,
+          return_to: return_path(@socket, @menu, @params),
+          pid_link_builder: &process_info_path(@socket, &1, @params) %>
+      <% end %>
+
+      <div class="card processes-card mb-4 mt-4">
+        <div class="card-body p-0">
+          <div class="processes-table-wrapper">
+            <table class="table table-hover mt-0 processes-table clickable-rows">
+              <thead>
+                <tr>
+                  <th class="pl-4">PID</th>
+                  <th>Name or initial call</th>
+                  <th class="text-right">
+                    <%= sort_link(@socket, @params, :memory, "Memory") %>
+                  </th>
+                  <th class="text-right">
+                    <%= sort_link(@socket, @params, :reductions, "Reductions") %>
+                  </th>
+                  <th class="text-right">
+                    <%= sort_link(@socket, @params, :message_queue_len, "MsgQ") %>
+                  </th>
+                  <th>Current function</td>
                 </tr>
-              <% end %>
-            </tbody>
-          </table>
+              </thead>
+              <tbody>
+                <%= for process <- @processes, list_pid = encode_pid(process[:pid]) do %>
+                  <tr phx-click="show_info" phx-value-pid="<%= list_pid %>" phx-page-loading class="<%= row_class(process, @pid) %>">
+                    <td class="processes-column-pid pl-4"><%= list_pid %></td>
+                    <td class="processes-column-name"><%= format_name_or_initial_call(process[:name_or_initial_call]) %></td>
+                    <td class="text-right"><%= process[:memory] %></td>
```

Comment:
there's a nice little function we can use to make it prettier:

```suggestion
                    <td class="text-right"><%= SystemInfo.format_bytes(process[:memory]) %></td>
```

we could even add a:

```elixir
def display_bytes(bytes) do
  content_tag(:span, format_bytes(bytes), title: bytes)
end
```

if someone wants the exact value.